resources/shaders/shadow_ray.hlsl

```diff
@@ -10,7 +10,7 @@ struct Attributes { };
 
 bool TraceShadowRay(uint idx, float3 origin, float3 direction, float far, unsigned int depth)
 {
-#ifndef NO_SHADOWS
+//#ifndef NO_SHADOWS
```

Comment:
Conflicts with this and your previous branch; if shadows can be disabled runtime already, this should be completely removed instead of commented. If they can only be disabled compile time, please leave this in (for benchmarking etc.).